### PR TITLE
bot: Add support postrun option

### PR DESCRIPTION
**Pull request by tedd-an**
This patch adds a support for postrun option which executes the scripts
after running the tests.

Signed-off-by: Tedd Ho-Jeong An <tedd.an@intel.com>

### CODE REVIEW

File: bot/config.py.sample [hunk 1]
```diff
@@ -115,4 +115,12 @@ z['scheduler'] = {
     'friday': '20:00',
 }
 
+# ****************************************************************************
+# Post run configuration
+#
+# To do any post run tasks.
+# ****************************************************************************
+z['postrun'] = ['/path/to/script params1 params2',
+		'/path/to/script2 params1 params2 param3']
```
Comment (MariuszSkamra) [inline bot/config.py.sample]: align

**Comment by kolodgrz** [comment]
@tedd-an do you have any specific scenario with usage of this postrun ?
I'm wondering if it'll be sufficient to introduce importable postrun iut specific module ("*.py" as is done for config.py). You may develope for all (public pushed to autopts) or have your own (private on disk and your personal use) Do we need another separate process ? I see you start process and next script command is just waiting for its end.

**Comment by tedd-an** [comment]
> @tedd-an do you have any specific scenario with usage of this postrun ?
> I'm wondering if it'll be sufficient to introduce importable postrun iut specific module ("*.py" as is done for config.py). You may develope for all (public pushed to autopts) or have your own (private on disk and your personal use) Do we need another separate process ? I see you start process and next script command is just waiting for its end.

**Comment by tedd-an** [comment]
@kolodgrz The main use case for postrun was to upload the test result to TestRail (https://zephyrproject.testrail.io/index.php?/projects/overview/6) which uses the script from here(https://github.com/zephyrproject-rtos/qm/blob/master/scripts/test_reports/report.py).
Could you be more specific about
>I'm wondering if it'll be sufficient to introduce importable postrun iut specific module ("*.py" as is done for config.py)
It can be imported as a module but don't want to make it a public code since not everyone can access the site and it is sort of internal things.

**Comment by tedd-an** [comment]
Discard the PR at this ponit.

**Pull request closed without merging**